MAUI/DataGrid/pull-to-refresh.md

```diff
@@ -10,6 +10,135 @@ keywords : maui datagrid, maui grid, grid maui, maui gridview, grid in maui, .ne
 
 # Pull To Refresh in MAUI DataGrid (SfDataGrid)
 
+The data grid enables the `PullToRefresh` option by setting the [SfDataGrid.AllowPullToRefresh]() property to `true`, the control supports for refreshing the data source at runtime while doing the pull to refresh action. 
+
+## Pull to refresh command
+
+The data grid refreshes the data in view at runtime by triggering an `ICommand` bound to the [SfDataGrid.PullToRefreshCommand]() property. When you perform a pull-to-refresh action, if the progress bar reaches 100%, this command is triggered to refresh the records in view.
+
+Set the [SfDataGrid.IsBusy]() property to `true` before refreshing the records to notify the grid that a pull-to-refresh action is being performed, and set the property to `false` after the view is refreshed. You can also adjust the pull-to-refresh animation time by setting a delay in the sample.
+
+To enable and perform the pull-to-refresh operation, follow the code example:
+
+{% tabs %}
+
+{% highlight Xaml %}
+
+<syncfusion:SfDataGrid x:Name="dataGrid"
+                       AllowPullToRefresh="True"
+                       PullToRefreshCommand="{Binding RefreshCommand}"
+                       ItemsSource="{Binding OrdersInfo}">
+{% endhighlight %}
+
+{% highlight c# %}
+
+dataGrid.AllowPullToRefresh = true;
+
+Command RefreshCommand = new Command(ExecutePullToRefreshCommand);
+dataGrid.PullToRefreshCommand = RefreshCommand;
+ 
+private async void ExecutePullToRefreshCommand()
+{
+    this.dataGrid.IsBusy = true;
+    await Task.Delay(new TimeSpan(0, 0, 5));
+    viewModel.ItemsSourceRefresh();
+    this.dataGrid.IsBusy = false;
+}
+
+//ViewModel.cs
+public void ItemsSourceRefresh()
+{
+    int count = random.Next (1, 10);
+    for (int i = 1; i <= count; i++) 
+    {
+        this.OrdersInfo!.Insert(0, new OrderInfo()
+        {
+            OrderID = i,
+            CustomerID = this.customerID[this.random.Next(15)],
+            EmployeeID = this.random.Next(1700, 1800),
+        });
+    }        
+}
+
+{% endhighlight %}
+{% endtabs %}
+
+<img alt="Pull To Refresh" src="Images\pull-to-refresh\net-maui-pull-to-refresh-slide-on-top.gif" width="404"/>
+
+### Transition Mode
+
+The [TransitionMode]() property specifies the mode of the animations. It has two modes:
```

Comment:
To customize the progress view animation, adjust the `SfDataGrid.TransitionMode` property. By default, the transition mode is set to `SlideOnTop`.